src/pages/Transportation/Register/Hajj.tsx

```diff
@@ -1,3 +1,91 @@
+import { Header } from "../../../components/header";
+import hajj1 from "../../../assets/Transportation/register/umrah-min.webp";
+import hajj2 from "../../../assets/Transportation/register/umrah_2.webp"
+import hajj3 from "../../../assets/Transportation/register/umrah_3.webp";
+import hajj4 from "../../../assets/Transportation/register/umrah_4.webp";
+import hajj5 from "../../../assets/Transportation/register/umrah_5.webp";
+import hajj6 from "../../../assets/Transportation/register/umrah_6.webp";
+
+import { RegisterCard } from "../../../components/Rigester_card";
+
 export function Hajj() {
-  return <div>Hajj</div>;
+  return (
+    <>
+      <Header heading="Hajj and Umrah groups" img="headerBg" />
+      <section className="">
+
+        <div className="flex gap-5 mb-8">
+        <RegisterCard
```

Comment:
please make an array of objects for the data. like

const HajjDestinationsData = [
{
heading: "add heading",
img: "add image",
description: "details about card",
date: " ",
date1: " ",
date2: " ",
price: " ",
rate: " ",
button: " ",
}
]

and then map this array render the RegisterCard component.